### ISSUE TITLE
Status of the project

### ISSUE THREAD

**Issue by auser** [aofdev/yaml-peg#1]
Curious if this project is still active or has it been abandoned?

**Comment by auser** [comment]
Love love love the state of the project. I've been taking some ideas from it and applying it to a project of my own. I'd love to get a few more components of the YAML spec in. In particular, objects 🥶

Any desire to bang through the code and try to get them in with me? Happy to collaborate

**Comment by aofdev** [comment]
> Curious if this project is still active or has it been abandoned?

Thank you for your interest in this project. I did it to learn PEG while also having fun.

**Comment by auser** [comment]
Yeah -- I've been into PEG for a bit... I'm a tad stumped on getting objects working, but other than that, it's flying by

**Comment by aofdev** [comment]
> Love love love the state of the project. I've been taking some ideas from it and applying it to a project of my own. I'd love to get a few more components of the YAML spec in. In particular, objects 🥶
> 
> Any desire to bang through the code and try to get them in with me? Happy to collaborate

Wow, I'd like to see your project ideas. enjoy collaborating

**Comment by auser** [comment]
Real-time or snail-mail time?

**Comment by aofdev** [comment]
> Real-time or snail-mail time?

I'm available by email.
